libethereum/BlockChainSync.cpp

```diff
@@ -209,24 +209,32 @@ void BlockChainSync::onPeerStatus(std::shared_ptr<EthereumPeer> _peer)
     std::shared_ptr<SessionFace> session = _peer->session();
     if (!session)
         return; // Expired
+    std::string disconnectReason;
```

Comment:
Use `char const*` not to construct `std::string` when the LOG is no going to be displayed anyway.